src/test/java/com/dumbdogdiner/stickyapi/common/webapis/CachedMojangAPITest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.webapis;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CachedMojangAPITest {
```

Comment:
Please used [Mojang Studios skin textures](https://minecraft.gamepedia.com/Head#Mojang_Studios_skins), as these usernames/uuids will never change in the future.